src/http/server/response.cr

```diff
@@ -182,6 +188,10 @@ class HTTP::Server
         end
       end
 
+      def closed?
+        @closed
+      end
+
```

Comment:
I guess this should be checked in `unbuffered_write`?

Reply:
`IO::Buffered#write` already calls `check_open`. This is ensured with the `expect_raises` expectation.